src/api.js

```diff
@@ -14,15 +14,18 @@ export async function getAnnouncements() {
     await axios.get(`${BASE_URL}/${ANNOUNCEMENTS}`);
 }
 
-export async function deleteAnnouncement(announcementId) {
+export async function putAnnouncements(announcementBody) {
+    await axios.put(`${BASE_URL}/${ANNOUNCEMENTS}`, { ...announcementBody });
+}
```

Comment:
You're modifying a single announcement not multiple so we have to specify the announcementId based on our REST spec.
```suggestion
export async function putAnnouncement(announcementId, announcementBody) {
    await axios.put(`${BASE_URL}/${ANNOUNCEMENTS}/${announcementId}`, { ...announcementBody });
}
```